### PR TITLE
Fix installation and organize notebooks

### LINKED ISSUE #4: Specify Python version, generate usable requirements.txt
* Need to add description on how to create a virtual env and what version of Python is required (3.11 seems to be okay)
* module not listed in required: agcounts
* requirements don’t specify what versions of packages should be installed; as a result, the default installation doesn’t work (starting from fresh conda env)
    > as above, then run:
    > ```python
    > import matplotlib.pyplot as plt
    > from pisces.experiments import DataSetObject
    > sets = DataSetObject.find_data_sets("../data_sets")
    > walch = sets['walch_et_al']
    > hybrid = sets['hybrid_motion']
    >  # => keras issue: Could not deserialize class 'Functional' because its parent module keras.src.engine.functional cannot be imported.

### PR DESCRIPTION
**Pull request by ftavella**
Closes #4. This PR gets rid of `requirements.txt` and relies on a handful of packages defined on `settings.ini` we can specify versions if we want to. 

Additionally, I created a command line tool called `pisces_setup` that should be run after `pip install` to convert the MO model to keras. This allows the package to be usable on Colab.

Also:
- All imports are now at the top to have a passing nbdev build
- Fixed a docstring error for `SimplifiablePrefixTree`
- Commented out the code that depended on `agcounts`
- Removed an import to `pkg_resources` on `data_sets` because it was causing an `etils` error

### CODE REVIEW

File: pisces/utils.py [hunk 1]
```diff
@@ -220,20 +220,16 @@ def max2epochs(data, fs, epoch):
...
 
 # %% ../nbs/00_utils.ipynb 11
-from agcounts.extract import get_counts
+# from agcounts.extract import get_counts
 
-def build_ActiGraph_official(time_xyz, axis: int = 3) -> Tuple[np.ndarray, np.ndarray]:
-    freq = 50
-    counts = get_counts(time_xyz[:, 1:], freq=freq, epoch=15)[:, axis - 1]
-    times = np.linspace(time_xyz[0, 0], time_xyz[-1, 0], len(counts))
+# def build_ActiGraph_official(time_xyz, axis: int = 3) -> Tuple[np.ndarray, np.ndarray]:
```
Comment (ericcanton) [inline pisces/utils.py:225]: I actually [removed ag counts entirely](https://github.com/Arcascope/pisces/pull/9/commits/a513caee45c82cf11c8143de6a574af81caffeab) on my branch. So I'd say let's just cut it.

**Review by ericcanton — CHANGES_REQUESTED** [review]
just some small things

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -1,10 +1,92 @@
...
+  workflow_dispatch:
+    inputs:
+      pre:
+        description: 'Install prerelease nbdev/execnb from master?'
+        required: false
+        default: ''
+      version:
+        description: 'Version of python to set up'
+        required: false
+        default: '3.9'
```
[the remark this answers is not in the document]
Comment (ftavella) [reply .github/workflows/test.yaml:12]: You're totally right!

**Review by ericcanton — APPROVED** [review]
nice work! thanks!
